packages/babel-parser/src/plugins/estree.js

```diff
@@ -379,16 +379,19 @@ export default (superClass: Class<Parser>): Class<Parser> =>
       return node;
     }
 
-    toReferencedListDeep(
-      exprList: $ReadOnlyArray<?N.Expression>,
-      isParenthesizedExpr?: boolean,
-    ): void {
+    finishCoverCallExpressionArguments(
```

Comment:
Trying to understand this method name. For my understanding, would you mind explaining it?

Reply:
It's just mirroring:

https://github.com/babel/babel/blob/4fe8c3acc5934268682d53714e9f42c49f7e8fcb/packages/babel-parser/src/parser/expression.js#L709

Ref: https://tc39.es/ecma262/#prod-CoverCallExpressionAndAsyncArrowHead

Reply:
The `Cover` prefix means that it is ambiguous: It can be a call expression _or_ an async arrow. In this case it seems to me at the time when `finishCoverCallExpressionArguments` is called, it is indeed a call expression.

However `finishCallExpressionArguments` is a bit confused, since we have called `finishCallExpression` before. Maybe just `toReferencedArguments`? (Naming is hard)

Reply:
> Naming is hard

Indeed it is, we also have `parseCallExpressionArguments` lol

Reply:
I renamed it `toReferencedArguments` as I struggled a bit with it, happy to rename again if someone can cook something up.

Reply:
Thanks for the explanation :+1: